lnbits/decorators.py

```diff
@@ -241,8 +276,8 @@ async def check_user_exists(usr: UUID4) -> User:
     return g().user
 
 
-async def check_admin(usr: UUID4) -> User:
-    user = await check_user_exists(usr)
+async def check_admin(req: Request, usr: Optional[str] = None) -> User:
```

Comment:
- why not use `UUID4` for `usr`?
```suggestion
async def check_admin(req: Request, usr: Optional[UUID4] = None) -> User:
```